blog/2025-01-14-build-3d-with-react.md

```diff
@@ -0,0 +1,97 @@
+---
+authors: mark
+slug: declarative-3d-with-playcanvas-react
+title: "Declarative 3D with React"
+tags:
+  - engine
+  - react
+---
+
+Since we introduced PlayCanvas, we've seen many developers around the world build amazing 3D experiences using the PlayCanvas Engine and Editor.
```

Comment:
```suggestion
Since introducing PlayCanvas, we've seen developers the world over build amazing 3D experiences using the PlayCanvas Engine and Editor. But what if you are a React developer?
```